bodhi/server/models.py

```diff
@@ -1248,6 +1305,43 @@ class Build(Base):
         'polymorphic_identity': ContentType.base,
     }
 
+    # Local cache in case we need more koji info
+    _kojiinfo = None
+
+    def _get_kojiinfo(self):
+        if not self._kojiinfo:
+            koji_session = buildsys.get_session()
+            self._kojiinfo = koji_session.getBuild(self.nvr)
+        return self._kojiinfo
+
+    def _get_n_v_r(self):
+        """Return the N, V and R components of an RPM build from NVR."""
```

Comment:
This isn't an RPMBuild, it's the Build superclass. Let's also document the return type. How about something like:

```
"""
Return the N, V, and R components for a traditionally dash-separated build.

Returns:
    tuple: A 3-tuple of name, version, and release.
"""
```